app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
```

Comment:
Place each import on a new line so there would not be imports on the same line with opening or closing parenthesis. It would help to improve the readability